### PR TITLE
Fix examples/cifer10_cnn.py error by missing `steps_per_epoch` parameter at model.fit_generator 

### PR DESCRIPTION
**Pull request by uchidama**
### Summary

examples/cifer10_cnn.py
This python script don't work because error.

ValueError: `steps_per_epoch=None` is only valid for a generator based on the `keras.utils.Sequence` class. Please specify `steps_per_epoch` or use the `keras.utils.Sequence` class.

Fix missing `steps_per_epoch` parameter at model.fit_generator

### PR Overview

- [n] This PR requires new unit tests [y/n] (make sure tests are included)
- [n] This PR requires to update the documentation [y/n] (make sure the docs are up-to-date)
- [y] This PR is backwards compatible [y/n]
- [n] This PR changes the current API [y/n] (all API changes need to be approved by fchollet)

### CODE REVIEW

File: examples/cifar10_cnn.py [hunk 1]
```diff
@@ -112,8 +113,8 @@
     datagen.fit(x_train)
 
     # Fit the model on the batches generated by datagen.flow().
-    model.fit_generator(datagen.flow(x_train, y_train,
-                                     batch_size=batch_size),
+    model.fit_generator(datagen.flow(x_train, y_train, batch_size=batch_size),
+                        steps_per_epoch=int(math.ceil(x_train.shape[0] / float(batch_size))),
```
Comment (abhinavsagar) [inline examples/cifar10_cnn.py]: Could you wrap `float(batch_size))),` to new line.

Comment (uchidama) [reply examples/cifar10_cnn.py]: Thanks, I added wrap.

https://github.com/keras-team/keras/pull/13019/commits/2550d0ca44d58027a34a714745855771acc59fe7

**Comment by fchollet** [comment]
The output of `flow` is in fact a `Sequence`, so the argument `steps_per_epoch` is not required. Can you check that your version of Keras is up-to-date?

**Comment by uchidama** [comment]
> The output of `flow` is in fact a `Sequence`, so the argument `steps_per_epoch` is not required. Can you check that your version of Keras is up-to-date?

Hi, @fchollet . Thanks for your reply. I bought a your keras book Japanese version.

When I runed cifer10_cnn.py code on google colaboratory, found this error.
I check keras version on google colaboratory by this code.

`import keras`
`print(keras.__version__) `
and this version number is printed.
`2.2.4`

**Comment by uchidama** [comment]
@fchollet , I made 2 scripts on google colab to compare(keras version is 2.2.4).

[cifer10_cnn code that steps_per_epoch is missing(now keras/examples code)](https://colab.research.google.com/drive/1dKgG_MZl3IYG8t2f171DzgJfX9M7lCr-)
 It can not train by ValueError.

> ValueError: `steps_per_epoch=None` is only valid for a generator based on the `keras.utils.Sequence` class. Please specify `steps_per_epoch` or use the `keras.utils.Sequence` class.

[cifer10_cnn code that step_per_epochs is added](https://colab.research.google.com/drive/1vf3F4b6YfWoOXDS-l48H-Bzlx23Gn6l3)
 It can train. no error.

**Pull request closed without merging**